web/src/nav/StatusMessagesNavItem.tsx

```diff
@@ -34,22 +44,32 @@ interface StatusMessageEntryProps {
     showLink?: boolean
     linkTo: string
     linkText: string
+    alert?: string
+    linkOnClick: (event: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => void
 }
 
-const StatusMessagesNavItemEntry: React.FunctionComponent<StatusMessageEntryProps> = props => (
-    <div key={props.text} className="status-messages-nav-item__entry">
-        <h4>{props.title}</h4>
-        <p>{props.text}</p>
-        {props.showLink && (
-            <p className="status-messages-nav-item__entry-link">
-                <Link to={props.linkTo}>Configure external services</Link>
-            </p>
-        )}
-    </div>
-)
+const StatusMessagesNavItemEntry: React.FunctionComponent<StatusMessageEntryProps> = props => {
+    let className = 'status-messages-nav-item__entry'
+    if (props.alert) {
+        className = `${className} alert ${props.alert} mb-0`
+    }
+    return (
+        <div key={props.text} className={className}>
```

Comment:
You could use `classNames()`:

```suggestion
        <div key={props.text} className={classNames('status-messages-nav-item__entry', props.alert && `alert ${props.alert} mb-0`)}>
```

Reply:
Ahh, good stuff!